besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -446,6 +450,31 @@ void setBannedNodeIds(final List<String> values) {
           "Require authentication for the JSON-RPC HTTP service (default: ${DEFAULT-VALUE})")
   private final Boolean isRpcHttpAuthenticationEnabled = false;
 
+  @Option(
+      names = {"--rpc-http-tls-enabled"},
+      description = "Enable TLS for the JSON-RPC HTTP service (default: ${DEFAULT-VALUE})")
+  private final Boolean isRpcHttpTlsEnabled = false;
+
+  @Option(
+      names = {"--rpc-http-tls-keystore-file"},
+      paramLabel = MANDATORY_FILE_FORMAT_HELP,
+      description =
+          "PKCS#12 format key store file for the JSON-RPC HTTP service. Required if TLS is enabled.")
+  private final Path rpcHttpTlsKeystoreFile = null;
+
+  @Option(
+      names = {"--rpc-http-tls-keystore-password-file"},
+      description =
+          "Key store password file for the JSON-RPC HTTP service. Required if TLS is enabled.")
```

Comment:
key store or keystore ?

Reply:
JDK documentation mentions it as one word "keystore" https://docs.oracle.com/javase/7/docs/api/java/security/KeyStore.html ... VertX mentions it as two words "key store" https://vertx.io/docs/apidocs/io/vertx/core/net/JksOptions.html